server/models/User.js

```diff
@@ -0,0 +1,13 @@
+import mongoose from "mongoose";
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+	email: {
+		type: String,
+		unique: true,
+		required: true
+	},
+	username: String
```

Comment:
Code looks great, but there is one potential issue:

Users will likely not log in, thus no point for a username. 
We will only require storing their emails for this project since those are already unique.

Reply:
The only reason the username is there is to be able to reference it in our emails.  For instance when a user signs up we can use it to say hello "username", welcome to Vindici-Co! Or something like that.  Is that something you'd like to have or should I remove it?

Reply:
We may also be able to have them fill out a **name** input box when they signup on the client-side rather than offloading it. @dylane1999 @DylanMorison